packages/tools/fetch-tool/src/fluidFetchArgs.ts

```diff
@@ -66,13 +67,34 @@ export function printUsage() {
     }
 }
 
+export function setArguments(values: {
+    saveDir: string,
+    paramURL: string
+    dumpMessages?: boolean,
+    dumpMessageStats?: boolean,
+    dumpSnapshotStats?: boolean,
+    dumpSnapshotTrees?: boolean,
+    overWrite?: boolean }) {
+    paramSaveDir = values.saveDir;
+    paramURL = values.paramURL;
+    dumpMessages = values.dumpMessages ?? dumpMessages;
+    overWrite = values.overWrite ?? overWrite;
+}
+
 export function parseArguments() {
     for (let i = 2; i < process.argv.length; i++) {
         const arg = process.argv[i];
         switch (arg) {
             case "--dump:rawmessage":
                 dumpMessages = true;
                 break;
+                case "--dump:rawmessage:overwrite":
+                dumpMessages = true;
+                overWrite = true;
+                break;
+            case "--overwrite":
```

Comment:
I don't think we need this case because we have raw message dump with and without overwrite. So what is this for?

Reply:
But we already have option for just snapshotTree fetch which is  dump:snapshotTree.